service/controller.go

```diff
@@ -1693,12 +1694,11 @@ func (s *service) getIpsFromAZNetworkLabel(ctx context.Context, nodeID, azNetwor
 		if match := pattern.FindStringSubmatch(key); len(match) == 4 {
```

Comment:
Comments above are incorrect now with updated matching:
- "Find the node label with matching AZNetwork" should be "Find the node label with IP that belongs to AZNetwork"
- "Found the node with the matching AZNetwork label, get its IP" should be "Found the node with IP that belongs to the AZNetwork"